packages/cli/lib/utils.js

```diff
@@ -266,3 +266,32 @@ exports.renameEJS = function() {
 
   return renameStream;
 };
+
+/**
+ * Get a validate function for object/array type
+ * @param {String} type 'object' OR 'array'
+ */
+exports.validateStringObject = function(type) {
```

Comment:
This utility function needs a test of its own

Reply:
Added